src/main/java/TUM.java

```diff
@@ -0,0 +1,54 @@
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+public class TUM {
+    public static void greetToUsers() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm TUM");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+    public static void main(String[] args) {
+        List<Task> taskList = new LinkedList<>();
+        greetToUsers();
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
```

Comment:
```suggestion
        Scanner in = new Scanner(System.in);
        String line = in.nextLine();
```
Could be more effective